_posts/2016-08-27-Milestone-on-OSS-by-Default.markdown

```diff
@@ -0,0 +1,126 @@
+---
+layout: post
+title: "Helping the Web towards OSS by Default"
+date: 2016-08-27 12:17
+author: orta
+categories: [force, node, microgravity, js, JavaScript web]
+series: Open Source by Default
+---
+
+The main Artsy.net website for the desktop, [Force][force_gh], was our first Artsy application to open it's source code, [Craig][craig] and [Brennan][brennan] did it [back in 2014][force_oss]. Force's public offering laid the groundwork for the iOS OSS projects to come afterwards: [Eidolon][eidolon_oss], [Eigen][eigen_oss], [Energy][energy_oss] and [Emergence][emergence_oss].
+
+Force wasn't quite Open Source by Default though, it represented a _really_ important step for  Artsy's OSS perspective but was not the end goal. We were opening our source, but not opening our process.
```

Comment:
Might sound better switching the sentence structure, putting "though" at the beginning:

> Though Force wasn't quite Open Source by Default, it represented ...